### DISCUSSION TITLE
Difference between make -j4 and make -j8

### DISCUSSION THREAD

**Discussion by npaloskey** [idaholab/moose#20634] [Q&A General] [answered]
Hello!

I was wondering what the difference between the two commands "make -j4" and "make -j8" are? In some of the tutorials it instructs you to use make -j4 where as in others it instructs you to use make -j8 to compile.

Thanks!

**Comment by GiudGiud** [comment, marked as answer]
Hello

The number after `-j` is the number of processes used to build the code. So if you have at least 8 CPU cores on your computer, `-j8` should be almost twice as fast as `-j4`. If you only have 4 CPU cores, then they should be the same roughly. 

Guillaume
